behaviors/README.md

```diff
@@ -231,6 +231,18 @@ _groups:
       ifEmpty: title
 ```
 
+Placeholders may check two different fields with a logical operator. This is handy for components with editable links, as you'll usually want to display a placeholder when _either_ the url or the link text are empty. Operators are case-insensitive, and you can use `AND`, `OR`, or `XOR`.
```

Comment:
Is this functionality limited to 2 fields or is it 2 or more with logical operators between each pair?